Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apt-get -qq update && apt-get install --no-install-recommends -y apt-transpo
               python-virtualenv \
               sudo \
     && \
-    pip install --upgrade pip && \
+    curl https://bootstrap.pypa.io/get-pip.py | python && \
```

Comment:
I have to say that I feel much more confident with pip-builtin methods for installation, 
which might include checksumming etc. rather than executing py code from *.io domains. I guess this line is a recommended best practive by pip community ? Then I'd be fine here.

Reply:
I had done it like this because in my testing I found that `pip install --upgrade pip` had no effect when trying to upgrade pip 9 to pip 10 during my local testing in minikube. I had the same issue on my Mac, where pip could not upgrade itself using pip.

This might not be the best way to do it (if anyone else wants to interject with another opinion), but it works better like this when I tested things.

Reply:
I've reverted this since it's still `pip install --upgrade` in the `move_to_18.01` PR.